### ISSUE TITLE
Choose what data to extract

### ISSUE THREAD

**Issue by Spaceface16518** [Spaceface16518/strat-collector#2]
The `streamPost` gets tons of data about posts, but what data gets extrapolated needs to be decided. This depends on the future vision for this project, which is in @vorana's wheelhouse.

**Comment by Spaceface16518** [comment]
As of commit a7e96ba, the data that is being extracted is just:

- Title
- Body text
- author
- tags if they exist

**Comment by vorana** [comment]
we also need to extract:
- Ups
- Downs
- Gilded
- MISC (consult together for this)

**Comment by Spaceface16518** [comment]
@vorana I am working on all of those except for

>Gilded

I have not yet figured out the correct key for it, but as soon as I test it I will find out. The changes will most likely be added to the [export branch](https://github.com/Spaceface16518/strat-collector/tree/export).

**EDIT: I also am adding a `netUps` property to `data` for easy access.**

**Comment by Spaceface16518** [comment]
Because of mongo, we don't really have to extract any data. We can just send the data straight from reddit into the database.

**Comment by vorana** [comment]
No, remember we want to format the data before we put it in that database. Will make it much easier later.
